offer-manager-domain/src/main/groovy/life/qbic/business/projects/create/CreateProjectDataSource.groovy

```diff
@@ -0,0 +1,31 @@
+package life.qbic.business.projects.create
+
+import life.qbic.business.exceptions.ApplicationDeniedException
+import life.qbic.business.exceptions.DatabaseQueryException
+
+/**
+ * Provides access to the project management datasource
+ *
+ * This interface collects methods to interact with the project management
+ * datasource in the context of the Create Project use case.
+ *
+ * @since 1.0.0
+ */
+interface CreateProjectDataSource {
+
+    /**
+     * Creates a new QBiC project in the data management platform.
+     *
+     * @param projectApplication A project application with the information necessary for the
+     * project registration
+     *
+     * @return Information about the created project
+     *
+     * @throws ApplicationDeniedException If the application was denied. Reasons for denial are
+     * currently:
+     *  1. A project with the same project title already exists
+     * @throws DatabaseQueryException If any technical interaction with the data source fails
+     */
```

Comment:
```suggestion
    /**
     * Creates a new QBiC project in the data management platform.
     *
     * @param projectApplication A project application with the information necessary for the
     * project registration
     *
     * @return Information about the created project
     *
     * @since 1.0.0
     * @throws ApplicationDeniedException If the application was denied. Reasons for denial are
     * currently:
     *  1. A project with the same project title already exists
     * @throws DatabaseQueryException If any technical interaction with the data source fails
     */
```

Reply:
huh, what changed?

Reply:
ah, nvm, Mr. Since was missing